src/app/files.js

```diff
@@ -107,6 +107,47 @@ function Files (storage) {
     return files
   }
 
+  //
+  // Tree model for files
+  // {
+  //   'a': { }, // empty directory 'a'
+  //   'b': {
+  //     'c': {}, // empty directory 'b/c'
+  //     'd': { '/readonly': true, '/content': 'Hello World' } // files 'b/c/d'
+  //     'e': { '/readonly': false, '/path': 'b/c/d' } // symlink to 'b/c/d'
```

Comment:
@axic
So this would be basically your proposal, with the slight difference, that instead of having a
* `/content` or `/path` or no property at all in case of a directory
* there would instead always be a `""` property

This would simplify lookup.

1. There would be no need to iterate over the object to filter out `/properties`
2. I can always return the `""` property's value and treat it according to the `/type` property instead of switching between `/content` or `/path` and eventually more properties or the above mentioned filtering.